src/scenes/game.js

```diff
@@ -69,37 +70,32 @@ export default class GameScene extends Phaser.Scene {
             }
         });
 
-        // add the space bar input
-        spacebar = this.input.keyboard.addKey(Phaser.Input.Keyboard.KeyCodes.SPACE);
         // add the tap/click input
         this.input.on("pointerdown", this.jump, this);
 
-        scoreText = this.add.text(16, 16, "Score: 0", {
+        this.scoreText = this.add.text(16, 16, "this.scores.currScore: 0", {
```

Comment:
woops... this is more me doing a "find + replace all" but not realizing how many difference instances there were of "score". It should be "Score: 0" since that's how all games should start.